Sources/SWBCore/SpecImplementations/Tools/SwiftCompiler.swift

```diff
@@ -3600,35 +3597,18 @@ public func discoveredSwiftCompilerInfo(_ producer: any CommandProducer, _ deleg
 
         // Values we will parse.  If we end up not parsing any values, then we return an empty info struct.
         var swiftVersion: Version? = nil
-        var swiftlangVersion: Version? = nil
-        var clangVersion: Version? = nil
+        var swiftTag: String? = nil
         var swiftABIVersion: String? = nil
 
-        // Note that Swift toolchains downloaded from swift.org have a swiftc with a different version format than those built by Apple; the 'releaseVersionRegex' reflects that format.  c.f. <rdar://problem/34956869>
-        let versionRegex = #/Apple Swift version (?<swiftVersion>[\d.]+) \(swiftlang-(?<swiftlangVersion>[\d.]+) clang-(?<clangVersion>[\d.]+)\)/#
-        let releaseVersionRegex = #/(?:Apple )?Swift version (?<swiftVersion>[\d.]+) \(swift-(?<swiftlangVersion>[\d.]+)-RELEASE\)/#
-        let developmentVersionRegex = #/Swift version (?<swiftVersion>[\d.]+)-dev \(LLVM (?:\b[0-9a-f]+), Swift (?:\b[0-9a-f]+)\)/#
+        let versionRegex = #/Swift version (?<swiftVersion>[\d.]+).*\((?<swiftTag>.*)\)/#
```

Comment:
Just confirming: works for both Xcode and swift.org toolchains?

Reply:
Yes. The different versions we have are:
```
Apple Swift version 6.1 (swiftlang-6.1.x.x.x clang-1x00.x.x.x) # apple release
Swift version 6.1 (swift-6.1-RELEASE) # swift.org release
Swift version 6.1-dev (LLVM hash, Swift hash) # swift.org nightlies and at desk
```

And I'm about to put up a PR to make it:
```
Swift version 6.1 (swift-6.1-DEVELOPMENT-SNAPSHOT-year-month-day-a) # nightlies
Swift version 6.1-dev (Swift hash LLVM hash) # at desk
```